packages/experimental/NativeDatePicker/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui-react-native/experimental-native-date-picker",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
These types of updates should be handled by beachball btw. You should never have to manually do it.

Reply:
I don't want any trouble with the bot labor union... reverted.